### DISCUSSION TITLE
Support available for managing versions for two separate sets of documentation?

### DISCUSSION THREAD

**Discussion by chipsenkbeil** [squidfunk/mkdocs-material#5651] [Q&A]
My website is in the process of being built out with mkdocs material, and I have the need to support two different sets of documentation on the same website: a library/CLI and a neovim plugin.

These versions are distinct from each other, and I'd like to be able to reference a stable version of each and also support versioning of each.

For example, https://distant.dev/cli/0.20.x and https://distant.dev/neovim/0.3.x would be two separate projects housed under one mkdocs repository.

Is this something that's possible with the `mike` provider today? Or using a custom provider like mentioned in https://github.com/squidfunk/mkdocs-material/issues/5568?

**Comment by squidfunk** [comment]
I think you'd need to build two different documentation projects if you want to version them separately. Maybe talk to the authors of mike what's the best approach here, but separating those projects is likely the best and most scalable alternative.

**Comment by chipsenkbeil** [comment]
Asking there to see if mike has any solution available: https://github.com/jimporter/mike/issues/162.

My next thought is to just do this manually by creating folders like discussed in https://github.com/squidfunk/mkdocs-material/issues/5568, but I wouldn't be able to take advantage of the versioning support that mkdocs material offers.
